src/quickstart/features/assistant.md

```diff
@@ -0,0 +1,9 @@
+# Assistant
```

Comment:
Can we flatten the file structure and move all the files in `features`, directly under `quickstart` ?

`src/quickstart/assistant.md` instead of `src/quickstart/features/assistant.md`